### PR TITLE
fix: seamos type fixes

### PR DESCRIPTION
**Pull request by dawnho**
Noticed some type failures after merging my last pr. fixing them here

Should probably add `build:docs` to the PR checks

### CODE REVIEW

File: src/seam-os/client.ts [hunk 1]
```diff
@@ -180,6 +182,7 @@ export class SeamOS {
   public readonly buildings = {
     create: this._postGrab("/buildings/create", "building"),
     get: this._getGrab("/buildings/get", "building"),
+    get_counts: this.get("/buildings/get_counts"),
```
Comment (dawnho) [inline src/seam-os/client.ts:185]: I couldn't use `_getGrab` here. Should I be using `.get` instead?

Comment (seveibar) [reply src/seam-os/client.ts:185]: You have to use _getGrab or the types will not work

Comment (seveibar) [reply src/seam-os/client.ts:185]: What error are you seeing?

**Comment by dawnho** [comment]
The type errors are here: https://github.com/seamapi/javascript/actions/runs/3925412653

File: src/seam-os/client.ts (hunk 1 above)
Comment (dawnho) [reply src/seam-os/client.ts:185]: See here:
Screen Shot 2023-01-15 at 7 35 53 PM

Comment (seveibar) [reply src/seam-os/client.ts:185]: the `/access_passes/get_counts` route on the backend is incorrect and should be fixed. It should expose `{ access_pass_counts }`

Comment (seveibar) [reply src/seam-os/client.ts:185]: all the counts endpoints seem to have this issue

Comment (seveibar) [reply src/seam-os/client.ts:185]: I'm investigating this

Comment (dawnho) [reply src/seam-os/client.ts:185]: kk, i'll leave this open until we address the backend issues. thx~

File: src/seam-os/routes.ts [hunk 2]
```diff
@@ -1161,6 +1149,52 @@ export interface Routes {
     formData: {}
     jsonResponse: {}
   }
+  "/access_passes/update/index": {
```
Comment (seveibar) [inline src/seam-os/routes.ts:1152]: hmm, this looks wrong

**Review by seveibar — APPROVED** [review]
can merge after correcting `/access_passes/update`

**Pull request merged**